src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestBase.cs

```diff
@@ -117,7 +134,7 @@ internal async Task PingPong(QuicConnection client, QuicConnection server)
 
         private QuicListener CreateQuicListener(QuicListenerOptions options) => new QuicListener(ImplementationProvider, options);
 
-        internal async Task RunClientServer(Func<QuicConnection, Task> clientFunction, Func<QuicConnection, Task> serverFunction, int iterations = 1, int millisecondsTimeout = 30_000, QuicListenerOptions listenerOptions = null)
+        internal async Task RunClientServer(Func<QuicConnection, Task> clientFunction, Func<QuicConnection, Task> serverFunction, int iterations = 1, int millisecondsTimeout = PassingTestTimeout, QuicListenerOptions listenerOptions = null)
```

Comment:
Here `millisecondsTimeout` is an iteration timeout, not a full test timeout. Is that ok?

Reply:
I think this should be ok. This is just default and the default is also for 1 iteration. It so happened this the value was same so I used to await hardcoded numbers.